health/middlewares.js

```diff
@@ -0,0 +1,44 @@
+const nHealth = require('n-health');
+
+const { HEALTCHECK_DUMMY_USER_UUID } = require('config');
```

Comment:
Where does this config live?

Reply:
This is the dependency that the original authors of the repo use to get environment variables. I've added this to Vault

Reply:
ok!